packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/relay/getAllowedMints.ts

```diff
@@ -0,0 +1,13 @@
+import { initializeDiscoveryDb } from '@pedalboard/basekit'
+import { config } from '../../config'
+import type { ArtistCoins } from '@pedalboard/storage'
+
+const usdcMintAddress = config.usdcMintAddress
```

Comment:
nit: not really needed

Reply:
It is!

Reply:
I thought you meant the line wasn't needed - yeah we could use the config directly